File: programovani/ulohy/files/modelovani-obrazcu-i/ctverec.cpp
#include <iostream>
using namespace std;


int main() {

    // Deklarace a inicializace
    const int vel = 4; // Inicializace velikosti ctverce

    // Cyklus pro namalovani obrazce
    for(int i = 1; i <= vel * vel; i++) {

        // Vypiseme hvezdicky
        cout << "* ";

        // Pokud jsme dosli na konec radky 
        if(i % vel == 0)

            // Vypiseme novou radku
            cout << endl;

    }

}

File: programovani/ulohy/files/modelovani-obrazcu-i/ctverec.py
# Deklarace a inicializace
vel = 4 # Inicializace velikosti ctverce

# Vnitrni cyklus pro jednotlive hvezdicky
for j in range(1, vel * vel + 1):           

    # Vypiseme hvezdicky
    print("* ", sep="", end="")

    # Pokud jsme dosli na konec radky 
    if(j % vel == 0):

        # Vypiseme novou radku    
        print(end="\n")